bots/pr/src/main/java/org/openjdk/skara/bots/pr/PullRequestCheckIssueVisitor.java

```diff
@@ -236,7 +236,7 @@ public void visit(WhitespaceIssue e) {
     @Override
     public void visit(MessageIssue issue) {
         var message = String.join("\n", issue.commit().message());
-        throw new IllegalStateException("Incorrectly formatted commit message: " + message);
+        addFailureMessage(issue.check(), "Incorrectly formatted commit message: " + message);
```

Comment:
I don't think a multiline value will format well as a failure message. The commit message is also printed in the status comment, so probably don't need to include it here. Maybe it would be a good idea to log the full message just in case.